pkgs/http/lib/src/browser_client.dart

```diff
@@ -27,18 +31,20 @@ BaseClient createClient() {
 }
 
 /// A `package:web`-based HTTP client that runs in the browser and is backed by
-/// [XMLHttpRequest].
+/// `window.fetch`.
+///
+/// This client inherits some limitations of `window.fetch`.
+/// - [BaseRequest.persistentConnection];
```

Comment:
Do you mean that this is ignored?

Reply:
Yeah, I started writing a comment then, then went to check the doc if we can implement the necessary behavior, but then forgot what I was doing :) Fixed it now.